frontend/controller/actions/identity.js

```diff
@@ -782,18 +782,22 @@ export default (sbp('sbp/selectors/register', {
   },
   'gi.actions/identity/fetchChatRoomUnreadMessages': async () => {
     const { ourIdentityContractId } = sbp('state/vuex/getters')
-    return (await sbp('chelonia/kv/get', ourIdentityContractId, KV_KEYS.UNREAD_MESSAGES))?.data || {}
+    if (ourIdentityContractId) {
+      return (await sbp('chelonia/kv/get', ourIdentityContractId, KV_KEYS.UNREAD_MESSAGES))?.data || {}
+    }
```

Comment:
This action can now return either `undefined` or `{}` or `.data`.... did you mean to implement it this way?

Reply:
Oh, yeah. I made mistake in here. Nice catch. Thanks